protocols/virtualfund/virtualfund.go

```diff
@@ -38,17 +38,8 @@ type GuaranteeInfo struct {
 	GuaranteeDestination types.Destination
 }
 type Connection struct {
-	Channel          *channel.TwoPartyLedger // todo: #420 deprecate in favor of
-	ConsensusChannel *consensus_channel.ConsensusChannel
-	GuaranteeInfo    GuaranteeInfo
-}
-
-// ledgerChannelAffordsExpectedGuarantees returns true if, for the channel inside the connection, and for each asset keying the input variables, the channel can afford the allocation given the funding.
-// The decision is made based on the latest supported state of the channel.
-//
-// Both arguments are maps keyed by the same asset.
-func (c *Connection) ledgerChannelAffordsExpectedGuarantees() bool {
-	return c.Channel.Affords(c.getExpectedGuarantees(), c.Channel.OnChainFunding) // todo: #420 deprecate
+	Channel       *consensus_channel.ConsensusChannel
+	GuaranteeInfo GuaranteeInfo
```

Comment:
The changes made to this file are more or less forced by removing the TwoPartyLedger from a Connection.